consensus/Cargo.toml

```diff
@@ -18,7 +18,7 @@ rand = { version = "0.8", default-features = false, features = ["std_rng"] }
 tokio = { version = "1", features = ["full"] }
 tracing-subscriber = "0.2"
 tracing = "0.1"
-bls12_381-bls = { version = "0.2" }
+bls12_381-bls = { version = "0.2", features = ["parallel"] }
```

Comment:
Could we specify in README/Documentation pages how many CPU cores are suitable to facilitate `parallel` execution?

Reply:
Which README/Documentation pages are you referring to?

Reply:
I would just add a comment to the toml file for now. We need a more comprehensive documentation which would make sense to add as a separate issue later on